tests/daemon/test-persistent-cache.c

```diff
@@ -1041,33 +1018,37 @@ test_persistent_cache_reads_cached_boot_offset (gboolean     *unused,
 }
 
 /*
- * Ensures that a request for the boot time offset doesn't cause a write to the
- * metadata file if the 'always_update_timestamps' parameter is set to FALSE.
+ * Ensures that after initialization the persistent cache will not read the
+ * metadata file again when calling get_boot_time_offset().
  */
 static void
-test_persistent_cache_get_offset_wont_update_timestamps_if_it_isnt_supposed_to (gboolean     *unused,
-                                                                                gconstpointer dontuseme)
+test_persistent_cache_reads_cached_boot_offset_when_initialized (gboolean *unused,
```

Comment:
As far as I can tell, the only difference between this test and `test_persistent_cache_reads_cached_boot_offset` is that this test unlinks the file on disk rather than mangling it. I feel like we probably don't need both tests, but if we do keep both, these tests should be named in a way that indicates their relationship to one another.